charts/ckan/values.yaml

```diff
@@ -68,9 +68,9 @@ ckan:
 
   nginx:
     image:
-      repository: public.ecr.aws/nginx/nginx
+      repository: nginx
```

Comment:
You probably do want to pull from Amazon's registry, so that you're not depending on Docker (i.e. a third-party service that wouldn't otherwise need to be up in order for your containers to start).